include/seqan/index/index_fm.h

```diff
@@ -205,6 +205,16 @@ class Index<TText, FMIndex<TSpec, TConfig> >
     typename Fibre<Index, FibreLF>::Type            lf;
     typename Fibre<Index, FibreSA>::Type            sa;
 
+    /*!
+     * @fn FMIndex::Index
+     * @brief Constructor
+     *
+     * @signature Index::Index();
+     * @signature Index::Index(text);
+     *
+     * @param[in] text The text to be indexed.
+     */
```

Comment:
so shall I change the documentation?
`Index::Index(revText)`
`revText   the reversed text to be indexed`

maybe with a link to the `reverse` function for strings?